### ISSUE TITLE
'entities/deleteAll' won't work if one or more entity contains no data

### ISSUE THREAD

**Issue by CinKon** [revolver-app/vuex-orm#30]
`dispatch('entities/deleteAll')` throws the following error, when there are empty entities:

```
TypeError: Cannot read property 'entity' of undefined
    at Store.deleteAll (vuex-orm.esm.js?e0cb:4928)
    at Array.wrappedActionHandler (vuex.esm.js?68ac:704)
    at Store.dispatch (vuex.esm.js?68ac:426)
    at boundDispatch (vuex.esm.js?68ac:332)
    at Store.resetState (actions.js?ebc3:73)
    at Array.wrappedActionHandler (vuex.esm.js?68ac:704)
    at Store.dispatch (vuex.esm.js?68ac:426)
    at Store.boundDispatch [as dispatch] (vuex.esm.js?68ac:332)
    at VueComponent.mappedAction (vuex.esm.js?68ac:880)
    at VueComponent.boundFn [as resetState] (vue.runtime.esm.js?40f7:189)
```

**Comment by kiaking** [comment]
Wow... hm... I'll look into this. Not sure why `entity` is undefined. It should always be there if the module are registered to database. Anyway this sounds very much like a bug.

**Comment by mastermunj** [comment]
Actual way of using deleteAll on all entities requires passing empty object. Unfortunately I could not find better way of doing it so.

```js
dispatch('entities/deleteAll', {})
```

@kiaking docs does not reflect these as they are removed in fixes you implemented post merge of PR #28 in https://github.com/revolver-app/vuex-orm/commit/bedf80016d7cb6cb465befc9b29a5a0b1f8f2203

**Comment by kiaking** [comment]
Aha. my fault then. So sorry about that. I'll look into it.

**Comment by kiaking** [comment]
I've made PR #31 to make `deleteAll('entities/deleteAll)` syntax work. @mastermunj May I ask you to review the PR when you have time? Test is passing but I wanted to make sure I'm not missing anything since it seems like you had some concern about implementation 🙏

**Comment by mastermunj** [comment]
LGTM. Please go ahead.

**Comment by kiaking** [comment]
This issue has been solved with Vuex ORM 0.13.1. Thank you all for your help! And sorry about the doc @mastermunj. I should have noticed it or at least tested...
